application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -309,6 +309,11 @@ public String getLogErrorChannelWebhook() {
         return logErrorChannelWebhook;
     }
 
+    /**
+     * Gets the config for the Starboard
+     *
+     * @return the config of the Starboard
+     * */
```

Comment:
shitty javadoc. improve. explain the feature. how would u explain ur grandma what the starboard feature does? or someone new to the project.

Reply:
That's exactly what we have used for `getHelpSystemConfig` so I thought I should do something similar, nevertheless, I will change it.

Reply:
then that javadoc is bad as well (im aware that i wrote it 😆)